scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -892,17 +886,18 @@ def _thread(handler, timeout, outdir, logfile, fifo_fn, pid_fn,
                         # of not enough CPU time scheduled by host for
                         # QEMU process during p.poll(this_timeout)
                         cpu_time = QEMUHandler._get_cpu_time(pid)
-                        if cpu_time < timeout and not out_state:
+                        if cpu_time < timeout and not _status:
```

Comment:
nit (here and at other similar places)
```suggestion
                        if cpu_time < timeout and _status is None:
```